daml-lf/spec/daml-lf-1.rst

```diff
@@ -685,9 +685,15 @@ available for usage::
       ::= 'no_key'
        |  'key' τ eₖ eₘ
 
+  Temeplate choice observers
+    ChObs
+      ::= 'no_observers'
+        | 'observers' eₒ
+
   Template choice definition
-    ChDef ::= 'choice' ChKind Ch (y : 'ContractId' Mod:T) (z : τ)  : σ 'by' eₚ ↦ e
+    ChDef ::= 'choice' ChKind Ch (y : 'ContractId' Mod:T) (z : τ) : σ 'by' eₚ ChObs ↦ e
```

Comment:
I would prefer to not speak about this detail in the spec. 
I mean I would be in favour making the choice observer always a list of parties and ignoring the special case where it is not defined and add a small comment in the `Serialization changes since version 1.6` section saying to choice from LF < 1.11 are interpreted as with empty choice observer. 

From the point of view of the specification this should be the same.

Reply:
Ok, that would be cleaner. I'll change it like that.